### PR TITLE
Created Data Models

### PR DESCRIPTION
**Pull request by injacmarko**

### CODE REVIEW

**Review by FilipZecevic — APPROVED** [review]

File: src/Data/Models/FuelLog.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+namespace Data
+{
+    public class FuelLog
+    {
+        public int Id;
+        public int Litres;
+        public int Cost;
```
Comment (FilipZecevic) [inline src/Data/Models/FuelLog.cs:8]: Currency?

File: src/Data/Models/FuelLog.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        public string TypeOfFuel;
```
Comment (FilipZecevic) [inline src/Data/Models/FuelLog.cs]: Enum...

File: src/Data/Models/Vehicle.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+namespace Data
+{
+    public class Vehicle
+    {
+        public int Id;
+        public string Model;
+        public string LicensePlate;
+        public string Color;
```
Comment (FilipZecevic) [inline src/Data/Models/Vehicle.cs:8]: Enum for color.

File: src/Data/Models/Vehicle.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+namespace Data
+{
+    public class Vehicle
+    {
+        public int Id;
+        public string Model;
```
Comment (FilipZecevic) [inline src/Data/Models/Vehicle.cs:6]: Enum.

File: src/Data/Models/FuelLog.cs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+namespace Data
+{
+    public class FuelLog
+    {
+        public int Id;
+        public int Litres;
```
Comment (FilipZecevic) [inline src/Data/Models/FuelLog.cs]: Other mesurment units?

**Review by FilipZecevic — CHANGES_REQUESTED** [review]

File: src/Data/Models/Vehicle.cs (hunk 3 above)
Comment (djordjevics) [reply src/Data/Models/Vehicle.cs:8]: dont need, it is fine to be string. 

up to us.. maybe we want shades of colors, enum would be bad for that. Maybe RGB, but it is too much for this practice.

File: src/Data/Models/FuelLog.cs (hunk 1 above)
Comment (injacmarko) [reply src/Data/Models/FuelLog.cs:8]: Dont need currency in the data layer as it will all be in euros here

**Review by djordjevics — APPROVED** [review]

**Review by djordjevics — APPROVED** [review]

**Pull request merged**